homeassistant/components/unifiprotect/repairs.py

```diff
@@ -0,0 +1,127 @@
+"""unifiprotect.repairs."""
+
+from __future__ import annotations
+
+from typing import cast
+
+from aiohttp import CookieJar
+from pyunifiprotect import ProtectApiClient
+import voluptuous as vol
+
+from homeassistant import data_entry_flow
+from homeassistant.components.repairs import ConfirmRepairFlow, RepairsFlow
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.aiohttp_client import async_create_clientsession
+from homeassistant.helpers.issue_registry import async_get as async_get_issue_registry
+
+from .const import (
+    CONF_ALL_UPDATES,
+    CONF_ALLOW_EA,
+    CONF_OVERRIDE_CHOST,
+    DEVICES_FOR_SUBSCRIBE,
+)
+
+
+class EAConfirm(RepairsFlow):
+    """Handler for an issue fixing flow."""
+
+    _hass: HomeAssistant
+    _api: ProtectApiClient
+    _entry: ConfigEntry
+
+    def __init__(
+        self, hass: HomeAssistant, api: ProtectApiClient, entry: ConfigEntry
+    ) -> None:
+        """Create flow."""
+
+        self._hass = hass
```

Comment:
Looks like the underlying base class already has `hass`

https://github.com/home-assistant/core/blob/6021cedb09a2b7d3434d307bc2c124c51c8f731a/homeassistant/data_entry_flow.py#L422

Reply:
Lol. Of course, it does. And I never even use mine.